.github/workflows/ci.yml

```diff
@@ -0,0 +1,132 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+
+  static:
+    name: Static checks
+    runs-on: ubuntu-16.04
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.12
+        id: go
+      
+      - name: Checkout code
+        uses: actions/checkout@master
+        
+      - name: Check dependency licenses
+        run: make license-check
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GOFLAG: -mod=readonly
+          GOPROXY: https://proxy.golang.org
+
+      - name: Check generated OpenAPI files
+        run: sha256sum -c client/SHA256SUMS > /dev/null || { echo "Please generate client code using make generate-client"; exit 1; }
+        
+  database:
+    name: Database checks
+    runs-on: ubuntu-16.04
+    services:
+      mysql:
+          image: mysql:5.7
+          ports:
+            - 13306:3306
+          env:
+              MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+      postgres:
+          image: postgres:9.6
```

Comment:
postgres does not define a health check so you have to do that yourself to get the container to work properly.  Here is an example of a couple of different usages https://github.com/actions/example-services/blob/master/.github/workflows/postgres-service.yml

Reply:
Cool, thanks!